### DISCUSSION TITLE
Validation rules

### DISCUSSION THREAD

**Discussion by cesarMtorres** [rappasoft/laravel-livewire-tables#1262] [Q&A]
hi there, is posible add validation rules like form request?

**Comment by lrljoe** [comment]
For what purpose/which field(s)?

**Comment by cesarMtorres** [comment]
Validate filters

**Reply by lrljoe** [reply]
Validation on the text inputs for example?

There's validation that runs internally within the component for various aspects, but I can certainly look at exposing the settings for that so that it's configurable.

**Reply by lrljoe** [reply]
This is in the roadmap for v3, current estimate is end of this month (October) or start of November.
